tasks/sep/007-fpac-upgrade/input.json

```diff
@@ -0,0 +1,176 @@
+{
+  "version": "1.0",
+  "chainId": "5",
+  "createdAt": 1707763323,
+  "meta": {
+    "name": "FPAC Upgrade Mk2 - Sepolia Testnet",
+    "description": "Upgrades the `DisputeGameFactory` and `OptimismPortal` proxy implementations, as well as the dispute games, anchor state registry, preimage oracle, and MIPS VM",
+    "txBuilderVersion": "1.16.3"
+  },
+  "transactions": [
+    {
+      "to": "0x189aBAAaa82DfC015A588A7dbaD6F13b1D3485Bc",
+      "value": "0",
+      "data": "0x9623609d00000000000000000000000005f9613adb30026ffd634f38e5c4dfd30a197fa100000000000000000000000054f8076f4027e21a010b4b3900c86211dd2c2deb00000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000000000000044abfdcced0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+      "contractMethod": {
+        "inputs": [
+          {
+            "internalType": "address",
+            "name": "_proxy",
+            "type": "address"
+          },
+          {
+            "internalType": "address",
+            "name": "_implementation",
+            "type": "address"
+          },
+          {
+            "internalType": "bytes",
+            "name": "_data",
+            "type": "bytes"
+          }
+        ]
+      },
+      "contractInputsValues": {
+        "_proxy": "0x05F9613aDB30026FFd634f38e5C4dFd30a197Fa1",
+        "_implementation": "0x54F8076f4027e21A010b4B3900C86211Dd2C2DEB",
```

Comment:
This one should not. We're upgrading to the storage setter first (verified here: https://sepolia.etherscan.io/address/0x54F8076f4027e21A010b4B3900C86211Dd2C2DEB#code), in order to update the owner of the `DisputeGameFactory` proxy.

This is because yesterday, the security team updated the `ProxyAdmin` owner on `op-sepolia`, which is the same address as the `DisputeGameFactory` owner currently. The current owner is the old multisig (now the mock foundation safe), and the new one is the 2/2 multisig between the council and the fnd.